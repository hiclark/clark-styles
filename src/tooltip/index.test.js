// @flow

import ToolTip from '.';

describe('ToolTip', () => {
  it('renders', () => {
    expect(ToolTip).toMatchSnapshot();
  });
});
