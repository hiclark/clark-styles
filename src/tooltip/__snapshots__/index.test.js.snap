// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolTip renders 1`] = `[Function]`;
